frontend/src/pages/FormStaffPage.vue

```diff
@@ -259,7 +170,7 @@ watch(formData, () => {
   if (matchError.value) {
     fuzzyBus.emit(undefined);
     resetSubmissionValidators();
-    revalidateBus.emit();
+    revalidateBus.emit(progressData[currentTab.value].fields);
```

Comment:
Looks like progressData[currentTab.value].fields will always be undefined...
Is it intended to be like this?

Reply:
Yes it is, undefined will force all fields visible on the screen to be validated. Is the same behavior as calling emit without params

Reply:
I'll remove the param here as it doesn't make sense anymore.